account_credit_control/wizard/credit_control_policy_changer.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2012-2017 Camptocamp SA
+# Copyright 2017 Okia SPRL (https://okia.be)
+# Copyright 2020 Manuel Calero - Tecnativa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class CreditControlPolicyChanger(models.TransientModel):
+    """ Wizard that is run from invoices and allows to set manually a policy
+    Policy are actually apply to related move lines availabe
+    in selection widget
+
+    """
+
+    _name = "credit.control.policy.changer"
+    _description = "Credit Control Policy Changer"
+
+    @api.model
+    def _default_move_lines(self):
+        """ Get default lines for fields move_line_ids
+        of wizard. Only take lines that are on the same account
+        and move of the invoice and not reconciled
+
+        :return: list of compliant move lines
+
+        """
+        context = self.env.context
+        active_ids = context.get("active_ids")
+        invoice_obj = self.env["account.move"]
+        if not active_ids:
+            return False
+        selected_lines = self.env["account.move.line"]
+        for invoice in invoice_obj.browse(active_ids):
+            if invoice.type in ("in_invoice", "in_refund", "out_refund"):
```

Comment:
```suggestion
            if invoice.type != "out_invoice":
```
Other types of journal entries that are not sales invoices are also those that are type 'entry', 'in_receipt', 'out_receipt'.

Reply:
Good observation

Reply:
@manuelcalerosolis , this conversation was marked as resolved, however I don't see this change has been done.